docs/manual.rst

```diff
@@ -207,7 +207,8 @@ General Attributes and Methods
   This can be thought of as a measure of the robustness of a geometry, where larger values of
   minimum clearance indicate a more robust geometry. If no minimum clearance exists for a geometry,
   such as a point, this will return `math.infinity`.
-
+  
+  `New in Shapely 1.7.1`
```

Comment:
This looks good to me. The one question I would have is whether we want the two requirement statements on one line:

![image](https://user-images.githubusercontent.com/34973839/104123742-ed466200-531a-11eb-912f-2d2870ceff2d.png)

Or on separate lines:

![image](https://user-images.githubusercontent.com/34973839/104123754-f9322400-531a-11eb-85db-972c52a60450.png)
 
To get the later, we'll need an *extra* new between them like this:

```rst
  `New in Shapely 1.7.1`

  Requires GEOS 3.6 or higher.
```

Reply:
On two lines might be nicer I think (in any case, otherwise there needs to be a punctuation (`.`) added at the end of this line)